mappings/net/minecraft/stat/Stats.mapping

```diff
@@ -1,27 +1,30 @@
 CLASS jl net/minecraft/stat/Stats
-	FIELD A PLAYER_KILLS Lvr;
-	FIELD B FISH_CAUGHT Lvr;
-	FIELD C MINE_BLOCK [Lvr;
-	FIELD a ID_MAP Ljava/util/Map;
-	FIELD b STATS Ljava/util/List;
-	FIELD f STARTED_GAME Lvr;
-	FIELD g WORLDS_CREATED Lvr;
-	FIELD h WORLDS_LOADED Lvr;
-	FIELD i MULTIPLAYER_JOINS Lvr;
-	FIELD j GAMES_LEFT Lvr;
-	FIELD k MINUTES_PLAYED Lvr;
-	FIELD l DISTANCE_WALKED Lvr;
-	FIELD m DISTANCE_SWUM Lvr;
-	FIELD n DISTANCE_FALLEN Lvr;
-	FIELD o DISTANCE_CLIMBED Lvr;
-	FIELD p DISTANCE_FLOWN Lvr;
-	FIELD q DISTANCE_DIVED Lvr;
-	FIELD r DISTANCE_BY_MINECART Lvr;
-	FIELD s DISTANCE_BY_BOAT Lvr;
-	FIELD t DISTANCE_BY_PIG Lvr;
-	FIELD u JUMPS Lvr;
-	FIELD w DAMAGE_DEALT Lvr;
-	FIELD x DAMAGE_TAKEN Lvr;
-	FIELD y DEATHS Lvr;
-	FIELD z MOB_KILLS Lvr;
+	FIELD A playerKills Lvr;
+	FIELD B fishCaught Lvr;
+	FIELD C mineBlock [Lvr;
+	FIELD E useItem [Lvr;
+	FIELD F breakItem [Lvr;
+	FIELD a idMap Ljava/util/Map;
+	FIELD b stats Ljava/util/List;
+	FIELD f startGame Lvr;
+	FIELD g createWorld Lvr;
+	FIELD h loadWorld Lvr;
+	FIELD i joinMultiplayer Lvr;
+	FIELD j leaveGame Lvr;
+	FIELD k playOneMinute Lvr;
+	FIELD l walkOneCm Lvr;
+	FIELD m swimOneCm Lvr;
+	FIELD n fallOneCm Lvr;
+	FIELD o climbOneCm Lvr;
+	FIELD p flyOneCm Lvr;
+	FIELD q diveOneCm Lvr;
+	FIELD r minecartOneCm Lvr;
```

Comment:
this name is amusing
(keep it, I'm just pointing out the name of this field is amusing)

Reply:
yes it is 11/10